kobo/apps/accounts/templates/socialaccount/signup.html

```diff
@@ -7,7 +7,11 @@
 
 <form class="registration registration--social" id="signup_form" method="post" action="{% url 'socialaccount_signup' %}">
   {# Friendly welcome message #}
+  {% if username %}
   <h1>{% blocktrans with username=form.username.value %}Welcome to KoboToolbox, {{username}}!{% endblocktrans %}</h1>
+  {% else %}
```

Comment:
If `username` is defined by the `blocktrans` tag, would this condition ever evaluate to true?